lib/partitions_validator_utils.pm

```diff
@@ -14,14 +14,61 @@ use scheduler 'get_test_suite_data';
 use testapi;
 use Test::Assert ':all';
 use Exporter 'import';
-our @EXPORT = 'validate_partition_table';
+our @EXPORT = qw(
+  validate_partition_table
+  validate_partition_creation
+  validate_filesystem
+  validate_read_write
+  validate_unpartitioned_space);
 
 sub validate_partition_table {
     my $args = shift;
     return if check_var('BACKEND', 's390x');    # blkid output does not show partition table for dasd
     record_info("Check $args->{table_type}", "Verify if partition table type is $args->{table_type}");
     my $table_type = (split(/\"/, script_output("blkid $args->{device}")))[-1];    # last element of output eg "gpt"
-    assert_equals($args->{table_type}, $table_type, "Partition table type does not correspond to the expected one.");
+    my $converter  = {msdos => 'dos'};
+    my $expected   = $args->{table_type};
+    $expected = $converter->{$args->{table_type}} if (exists $converter->{$args->{table_type}});
```

Comment:
minor thing, you could reuse `$expected` variable here instead of repeating `$args->{table_type}`

Reply:
done